manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -0,0 +1,213 @@
+{{- if .Values.enableArcExtension }}
+{{- if .Values.arc.enableMonitoring }}
+{{- if .Values.linux.enabled}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: arc-{{ template "sscdpa.fullname" . }}-monitoring
+  namespace: {{ .Release.Namespace }}
+{{ include "sscdpa.arc.labels" . | indent 2 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: "arc-{{ template "sscdpa.fullname" . }}"
+  template:
+    metadata:
+{{ include "sscdpa.arc.labels" . | indent 6 }}
+    spec:
+      serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
```

Comment:
Does this pod require host network? Could you add a comment here why this is required so we have it for future ref?

Reply:
removed hostNetwork